dashboard/test/ui/features/level_group_multi_page.feature

```diff
@@ -45,7 +45,7 @@ Scenario: Submit three pages.
   And I press ".submitButton:first" using jQuery
   And I wait to see ".modal"
   And I press ".modal #ok-button" using jQuery
-  And I wait for 5 seconds
+  And I wait for 3 seconds
```

Comment:
hopefully this won't increase flakiness.. but seemed alright to make these delays consistent.